playback/tape_cassette.py

```diff
@@ -151,6 +154,26 @@ def _match_metadata_value(match_value, recorded_value):
 
         return recorded_value == match_value
 
+    @staticmethod
+    def operator_filter(recorded_value, metadata_value):
+        """
+        Check if this is an operator metadata filter and its value is in range
+        """
+        result = False
+        if 'operator' in metadata_value and 'value' in metadata_value:
```

Comment:
What happens if the value is actually a dict that we want to compare and not an operator based operation.
This will return always False.

You should have this if part (line 163) added as and condition to line 146, so if this is a dict without these properties it will fall back to line 155. Also add a test for that

Reply:
Done